catroid/src/main/java/org/catrobat/catroid/ui/fragment/ProjectListFragment.java

```diff
@@ -392,15 +391,13 @@ public void run() {
 
 	@Override
 	public void onLoadProjectSuccess(boolean startProjectActivity) {
-		if (context == null && getActivity() == null) {
-			Log.e(TAG, "Null Value at onLoadProjectSuccess");
+		if (getActivity() == null) {
+			Log.e(TAG, "Null Value of getActivity() at onLoadProjectSuccess");
```

Comment:
You don't need to check for `null` because the Task which triggers this callback should have been canceled anyway.
If the problem still exists, Crashlytics will tell us.